uploadNeuroDB/NeuroDB/objectBroker/MriUploadOB.pm

```diff
@@ -71,7 +71,8 @@ use TryCatch;
 
 # These are the only fields modified when inserting a new MRI upload record
 my @MRI_UPLOAD_FIELDS = (
-    'UploadedBy' ,'UploadDate','TarchiveID','DecompressedLocation', 'UploadLocation', 'PatientName'
+    'UploadedBy' ,'UploadDate','TarchiveID','DecompressedLocation', 'UploadLocation', 'PatientName',
```

Comment:
When selecting by `tarchiveLocation`, the `PatientName` field is present in both `tarchive` and `mri_upload` tables which returns an SQL error that `PatientName` is present in both tables.

Reply:
weird that it did not come up before...